platform/admin/core/admin-core-jolokia/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -12,15 +12,16 @@
  **/ -->
 <blueprint xmlns="http://www.osgi.org/xmlns/blueprint/v1.0.0">
 
-    <bean id="jolokiaServlet" class="org.jolokia.osgi.servlet.JolokiaServlet">
-    </bean>
+    <bean id="jolokia" class="org.jolokia.osgi.servlet.JolokiaServlet"/>
 
-    <service ref="jolokiaServlet" id="jolokiaServletService" interface="javax.servlet.Servlet">
+    <service interface="javax.servlet.Servlet" ref="jolokia">
         <service-properties>
-            <entry key="alias" value="/admin/jolokia"/>
+            <entry key="osgi.http.whiteboard.servlet.pattern" value="/jolokia/*"/>
+            <entry key="osgi.http.whiteboard.servlet.name" value="jolokia"/>
+            <entry key="osgi.http.whiteboard.context.select" value="(osgi.http.whiteboard.context.path=/*)"/>
```

Comment:
The ddf-admin-ui apps register based on their Context-Paths. Previously pax-web/ jetty would still register all servlets under each context. Now any servlets that need to work with those WABs apps will need to register under /* for the context path (believe it defaults to just /).

Reply:
updated context path to `/admin`